src/docs/community/jean-thierry/README.md

```diff
@@ -0,0 +1,13 @@
+# In honor of Jean-Thierry Kéchichian
+
+![image](https://github.com/OrchardCMS/OrchardCore/assets/24724371/f73763e8-874d-443e-868a-ceaa104951c7)
```

Comment:
A bit better would be to add the image to the docs project.

Reply:
yes. I added the images to the project.